buildenv/jenkins/Jenkinsfile_personal

```diff
@@ -2,18 +2,18 @@
 pipeline {
     agent {label "${params.MACHINE_LABEL}"}
     parameters {
+    	string (defaultValue: "AdoptOpenJDK", description: 'personal repo', name: 'personalRepo')
+    	string (defaultValue: "master", description: 'personal branch', name: 'personalBranch')
 		choice (choices: 'x64_linux\nx64_mac\ns390x_linux\nppc64le_linux\naarch64_linux', description: 'PLATFORM?', name: 'PLATFORM')
 		choice (choices: 'SE90\nSE80', description: 'What is JAVA_VERSION?', name: 'JAVA_VERSION')
 		choice (choices: 'openjdk9-openj9\nopenjdk8-openj9\nopenjdk9\nopenjdk8', description: 'What is JVM Version?', name: 'JVM_VERSION')
-		string (defaultValue: 'runtest', description: 'Test Target, e.g.openjdk sanity extended test_systemtest test_DirectoryName', name: 'TARGET')
+		string (defaultValue: '', description: 'Specific test project,default is all project, e.g. openjdk_regression systemtest performance jck thirdparty_containers', name: 'TESTPROJECT')
```

Comment:
Minor rewording:
Specific test project, leave blank by default for all projects to be compiled, e.g. openjdk_regression systemtest performance jck thirdparty_containers

Reply:
Update, thanks!